src/shared/Microsoft.Git.CredentialManager.Tests/SettingsTests.cs

```diff
@@ -24,10 +23,10 @@ public void Settings_IsDebuggingEnabled_EnvarUnset_ReturnsFalse()
         [Fact]
         public void Settings_IsDebuggingEnabled_EnvarTruthy_ReturnsTrue()
         {
-            var envars = new EnvironmentVariables(new Dictionary<string, string>
+            var envars = new TestEnvironment
```

Comment:
In the interest of DRY, should we extract a helper function to set up the test environment?

Reply:
There is also the argument about being explicit in unit tests; being able to see what the preconditions and postconditions are for each test without jumping into several other methods.

Not sure what benefit this would do us here either since each test has a different setup with different variables.